internal/metrics/prometheus.go

```diff
@@ -152,6 +154,23 @@ func NewCtrlFuncMetrics() *CtrlFuncMetrics {
 		[]string{SuccessKey},
 	)
 
+	// TODO: add new metric for fallback config generation or add a "fallback" label?
+	controllerMetrics.TranslationDuration = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name: MetricNameTranslationDuration,
+			Help: fmt.Sprintf(
+				"Duration of translations from Kubernetes state to Kong state."+
+					"`%s` describes whether there were unrecoverable errors (`%s`) or not (`%s`). "+
+					"Unrecoverable error in this case means KIC wasn't able to translate a Kubernetes object to Kong model.",
+				SuccessKey, SuccessFalse, SuccessTrue,
+			),
+			// TODO: A simple configuration translation with 1 ingress and 1 service costs 100~200 microseconds.
+			// Should we use a smaller starting value of the buckets?
+			Buckets: prometheus.ExponentialBuckets(1, 2, 20),
```

Comment:
I don't believe we should cover that. In a more realistic scenario there will almost always be more than that in the cluster.